app/export/csv.js

```diff
@@ -46,6 +32,19 @@ const convertRow = (row) => {
     getMicrochip(row, 2),
     row.exported_date,
     row.stolen_date,
+    // Owner
```

Comment:
Can we take these comments out please? If you need to make it clear where each data item is from you could store in a separate variable and use spread operator to combine them.